lightning/src/ln/update_fee_tests.rs

```diff
@@ -62,7 +62,7 @@ pub fn test_async_inbound_update_fee() {
 		*feerate_lock += 20;
 	}
 	nodes[0].node.timer_tick_occurred();
-	check_added_monitors!(nodes[0], 1);
+	check_added_monitors(&nodes[0], 1);
```

Comment:
It's nice to de-macro, but doing it in a separate PR would've been fine too. Just pre-formatting is a lot already.

Reply:
No need to change anything now. Comment was directed at future work mostly.